egov-user-event/LOCALSETUP.md

```diff
@@ -0,0 +1,36 @@
+# Local Setup
+
+To setup the egov-user-event in your local system, clone the [Muncipal Service repository](https://github.com/egovernments/core-services).
+
+## Dependencies
+
+### Infra Dependency
+
+- [X] Postgres DB
+- [ ] Redis
+- [X] Elastic search
+- [X] Kafka
+  - [X] Consumer
+  - [X] Producer
+
+## Running Locally
+
+To run the egov-user-event locally, you need to port forward below services locally
+
+```bash
+function kgpt(){kubectl get pods -n egov --selector=app=$1 --no-headers=true | head -n1 | awk '{print $1}'}
+kubectl port-forward -n egov $(kgpt egov-persister) 8082:8080
```

Comment:
Why persister needs to be port forwarded?

Reply:
Updated the above command